packages/@aws-cdk/cdk/lib/core/tokens/cfn-tokens.ts

```diff
@@ -0,0 +1,52 @@
+import { ResolveContext, Token } from "./token";
+
+/**
+ * A Token that represents a CloudFormation reference to another resource
+ */
```

Comment:
Add some implementation documentation here that explains how cross stack references work